internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -276,48 +276,17 @@ func TestAccClusterAdvancedCluster_advancedConfig(t *testing.T) {
 		CheckDestroy:             acc.CheckDestroyCluster,
 		Steps: []resource.TestStep{
 			{
-				Config: configAdvanced(projectID, clusterName, processArgs, nil, nil),
+				Config: configAdvanced(projectID, clusterName, processArgs, nil),
 				Check:  checkAdvanced(clusterName, "TLS1_1", "-1"),
 			},
 			{
-				Config: configAdvanced(projectID, clusterNameUpdated, processArgsUpdated, conversion.IntPtr(100), nil),
+				Config: configAdvanced(projectID, clusterNameUpdated, processArgsUpdated, conversion.IntPtr(100)),
 				Check:  checkAdvanced(clusterNameUpdated, "TLS1_2", "100"),
 			},
 		},
 	})
 }
 
-func TestAccClusterAdvancedCluster_advancedConfig_MongoDBVersion5(t *testing.T) {
```

Comment:
q: I see we initially added this test to ensure new attributes not compatible with older MongoDB versions would not cause errors (https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2592). Should we rename this test to OldMongoDBVersion and transition over to version 6?

Reply:
in that link it looks like it's specific to 5.  "set this attribute if MongoDB version is less than 6.0."

Reply:
I think we can go ahead and delete this test, if compatible checks are not needed for 6